lib/content/message/last_trip/no_service.ex

```diff
@@ -1,33 +1,28 @@
 defmodule Content.Message.LastTrip.NoService do
-  @enforce_keys [:destination, :line]
-  defstruct @enforce_keys ++ [:route]
+  @enforce_keys [:destination]
+  defstruct @enforce_keys ++ [:route, variant: :long]
 
   @type t :: %__MODULE__{
           destination: PaEss.destination(),
           route: String.t() | nil,
-          line: :top | :bottom
```

Comment:
The 'line' pattern is also used in `custom.ex`. Should we go ahead and refactor that while we're at it?

Reply:
That one's a bit different because we can't programmatically shorten custom messages, like we can with these variants.